defaults/main.yml

```diff
@@ -25,3 +25,6 @@ postgres_user_gid: 121
 
 configure_docker: yes
 docker_package: docker-engine
+
+# install packages to interface with SLURM DRM
+galaxy_slurm_drm: true
```

Comment:
How about we match docker right above and call this ``configure_slurm`` instead?